tensor/datafs/metadata.go

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) 2024, Cogent Core. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package datafs
+
+import (
+	"cogentcore.org/core/base/errors"
+	"cogentcore.org/core/base/fsx"
+	"cogentcore.org/core/base/metadata"
+	"cogentcore.org/core/plot/plotcore"
+	"cogentcore.org/core/tensor/table"
+)
+
+// This file provides standardized metadata options for frequent
+// use cases, using codified key names to eliminate typos.
+
+// SetMetaItems sets given metadata for items in given directory
+// with given names.  Returns error for any items not found.
+func (d *Data) SetMetaItems(key string, value any, names ...string) error {
+	its, err := d.Items(names...)
+	for _, it := range its {
+		it.Meta.Set(key, value)
+	}
+	return err
+}
+
+// PlotColumnZeroOne returns plot options with a fixed 0-1 range
+func PlotColumnZeroOne() *plotcore.ColumnOptions {
```

Comment:
We want `datafs` to import `plotcore`?

Reply:
yes.  apparently doing "resolve" does NOT apply the comment. :(